deps/build.jl

```diff
@@ -0,0 +1,18 @@
+# Downloads the SFD98 Map file
+const SFD98_BASEURL = "http://sncosmo.github.io/data/dust/"
+
+destdir = haskey(ENV, "SFD98_DIR") ? ENV["SFD98_DIR"] : @__DIR__
```

Comment:
It shouldn't matter too much for performance here, but I think it's better to keep using the best practices when possible:
```suggestion
const destdir = haskey(ENV, "SFD98_DIR") ? ENV["SFD98_DIR"] : @__DIR__
```